backend/filldb.py

```diff
@@ -0,0 +1,290 @@
+import argparse
+import json
+import random
+import string
+import uuid
+from random import choice
+from typing import List, Optional
+
+from loguru import logger
+from oasst_backend.api.deps import api_auth, get_dummy_api_client
+from oasst_backend.api.v1.utils import prepare_conversation
+from oasst_backend.config import settings
+from oasst_backend.database import engine
+from oasst_backend.models import ApiClient, message_tree_state
+from oasst_backend.prompt_repository import PromptRepository, TaskRepository, UserRepository
+from oasst_backend.tree_manager import TreeManager, TreeManagerConfiguration
+from oasst_shared.schemas import protocol as protocol_schema
+from oasst_shared.schemas.protocol import User as ProtocolUser
+from pydantic import BaseModel
+from sqlmodel import Session
+
+
+class MockMessage(BaseModel):
+    task_message_id: str
+    user_message_id: str
+    parent_message_id: Optional[str]
+    text: str
+    role: str
+
+
+class FillDb:
+    """Class object which fills the database: Api Client, Users & Messages."""
+
+    def __init__(self, db_engine, num_api_clients: int = 10, num_users: int = 10, use_seed: bool = True, seed: int = 0):
```

Comment:
Should the default be something like 500K messages?
Or is it taking a lot of time?